File: node/Foo.js
function getFoo() {
  return "Foo!";
}

function fibonacci(n) {
  return n <= 1 ? 1 : this.fibonacci(n - 1) + this.fibonacci(n - 2);
};

class Person {

  constructor(firstName, lastName) {
    this.firstName = firstName
    this.lastName = lastName
  }

  fullName() {
    return this.firstName + ' ' + this.lastName
  }
}

module.exports = {fibonacci, getFoo, Person}

File: java/Foo.java
class Foo
{
	public static String getFoo()
	{
		return "Foo!";
	}

	public static int fibonacci(int n)
	{
		return n <= 1 ? 1 : fibonacci(n - 1) + fibonacci(n - 2);
	}
}

class Person
{
	String firstName;
	String lastName;

	public Person(String firstName, String lastName) {
		this.firstName = firstName;
		this.lastName = lastName;
	}

	public String fullName() {
		return this.firstName+' '+this.lastName;
	}

}